test/vvenc_unit_test/vvenc_unit_test.cpp

```diff
@@ -241,11 +282,14 @@ static bool test_TCoeffOps()
 
 int main()
 {
+  unsigned seed = rand();
+  srand( seed );
```

Comment:
I think this makes no sense. It gets a random number seeded with 1, and then seeds the random number generator with that "random" value, so it will always return the same sequence of random numbers. If you want to have a different sequence each time the tests run, you usually seed it with the current timestamp.

Reply:
The initial seed is always 1? Didnt know that. Changed to a `time` call. Thanks